sdk/servicebus/Azure.Messaging.ServiceBus/Azure.Messaging.ServiceBus.sln

```diff
@@ -31,12 +31,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{8B8C
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.Experimental", "..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj", "{5871D9C6-F2DF-4F05-B29A-6C0D8709784A}"
```

Comment:
Re-adding `Azure.Core` changed the ordering and GUIDs.   I'll push a revert to `master` once the test suite completes.